src/layouts/base-new/index.tsx

```diff
@@ -27,6 +33,9 @@ const BaseNewLayout: React.FC<BaseNewLayoutProps> = ({
 
   return (
     <>
+      <Head>
+        <link rel="prefetch" href={DATADOG_SCRIPT_URL} />
```

Comment:
Intent here is to prefetch the DataDog RUM script so that, if consent is given, the script will load more quickly.